xml/dm_registers.xml

```diff
@@ -175,6 +175,11 @@
         <field name="hartsel" bits="25:16" access="R/W" reset="0">
             The DM-specific index of the hart to select. This hart is always part of the
             currently selected harts.
+
+            When the written value does not specify a hart that exists in the
+            system, this field may take on any value (valid or invalid). Unless
```

Comment:
what is the meaning of an "invalid" value? Do you mean "it can take on the value of a nonexistent hart"?